pkg/session/session.go

```diff
@@ -34,7 +34,15 @@ const (
 	passwordFormKey = "password"
 )
 
-var ErrInvalidCredentials = errors.New("invalid username or password")
+type InvalidCredentialsError struct {
+	Username string
```

Comment:
Is this actually used anywhere? I only see it set when initializng ?
Wouldnt an empty struct be enough?

Reply:
I originally had this field included in the `Error` method, but I've taken it out for now (until we support multiple users). Having the field means that we can read the username from the calling code, so it's useful, just not currently in use.